### DISCUSSION TITLE
Unifying the theme

### DISCUSSION THREAD

**Discussion by RaphGL** [dracula/firefox#8] [General]
Since we now have a userChrome.css theme, I would like to unify both themes so if a user uses one and decides to switch to the other one the experience would be seamless. 

The themes are mostly equivalent but there are some differences:
- the search bar is dark purple on manifest but black on chrome (I would like to have both dark purple)
- the borders are purple on manifest and blue on chromecss, I'd rather use the blue from the chromecss on both versions
- there's a blue border on the search bar that becomes purple when the bar is active, I would like to make this the same on chromecss or the other way around, but since the theme already has over 7000 users on the firefox add-on store removing this might piss off some users so I'd probably rather keep it

Lastly, the rectangle on top of the current tab on the firefox debugger should if possible use the same blue colors as the accent color used throughout the theme on chromecss.

Let me know what you think, @jannikbuscha and whoever might read this.

**Comment by jannikbuscha** [comment]
I would definitely keep the Firefox addon theme because it is easy to install and there are some who don't want the hassle of the userChrome theme.

To your four points I would have the following feedback:
1. you use here the color #353956 / rgb(53, 57, 86). Where did you get it? I can't find it anywhere in the docs.
2. do you want to update it yourself in the manifest or should I do it?
3. Do you mean like this? I would also use the pink more as an accent color, which is what most people do.
![image](https://user-images.githubusercontent.com/74017697/227661046-5cd291c8-236c-4c07-88e5-2b4b7d739e58.png)
4. can you show me a screenshot of what you mean?

**Reply by RaphGL** [reply]
> you use here the color #353956 / rgb(53, 57, 86). Where did you get it? I can't find it anywhere in the docs.

Yeah I merely made the purple darker so it wouldn't pop out as much. But when unifying with yours I would go back to complying with the spec since you used the spec version.

> do you want to update it yourself in the manifest or should I do it?

I can update the manifest it's not much work

> Do you mean like this? I would also use the pink more as an accent color, which is what most people do.

I personally hate the pink accent color, it's one of the reasons why I can't stand a lot of the official themes. VSCode has the best dracula theme imo (they use the same colors as I do here, they were the inspiration for it in fact). But I don't mind having alternative versions with different accent colors.

> can you show me a screenshot of what you mean?

This is the accent color that is being used on the active tab (blue)
![image](https://user-images.githubusercontent.com/28673457/227716207-8a8931df-ed55-4996-8fd3-8ad3feecf798.png)
It should be using the accent color used here
![image](https://user-images.githubusercontent.com/28673457/227716178-c4628c63-9dcd-4077-85d5-526e93eced1d.png)
Or just being pink (that'd be fine too)

**Reply by jannikbuscha** [reply]
> Yeah I merely made the purple darker so it wouldn't pop out as much. But when unifying with yours I would go back to complying with the spec since you used the spec version.

I'll change it with the spec version.

> I can update the manifest it's not much work

I have updated it now. (You could update it in the firefox addon-store)

> I personally hate the pink accent color, it's one of the reasons why I can't stand a lot of the official themes. VSCode has the best dracula theme imo (they use the same colors as I do here, they were the inspiration for it in fact). But I don't mind having alternative versions with different accent colors.

I'll leave the pink in userChrome and the purple in the addon.

> This is the accent color that is being used on the active tab (blue)

I don't know how to change this unfortunately, the devtools seem to be really limited as far as css styling goes.


Do you know how to update the article on https://draculatheme.com/firefox?

**Reply by RaphGL** [reply]
> I have updated it now. (You could update it in the firefox addon-store)

Added a few things you missed. I'll be adding it soon.

> I'll leave the pink in userChrome and the purple in the addon.

That would mean that the themes are not unified though, if you want we can create variations on the theme or something.

> Do you know how to update the article on https://draculatheme.com/firefox?

That's automated, I don't know how often the bot looks for changes on here, but it eventually does.

**Reply by RaphGL** [reply]
I've updated the theme on the Firefox Addons store

**Reply by jannikbuscha** [reply]
> That would mean that the themes are not unified though, if you want we can create variations on the theme or something.

yes that's right, what's your suggestion? Pink is by itself the default color but by itself that wouldn't be a small change. Maybe a second addon theme?

**Reply by RaphGL** [reply]
I could get behind having an alternative version of the theme. one using pink (since it's a common enough accent color) and one using the purple/blue (comment color) I'm using and you used on the popups yourself.

The thing is, dracula was built originally for syntax highlighting and it has no proper spec on what to use on GUIs so everyone just made dracula themes to their own preferences, I got to like Dracula through using the VSCode theme and that's my base for all my dracula themes, other people might have gotten to like it from some other theme and adopted different colors from the spec.
